File: dotnet/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/WindowsDialog.cs
namespace AbstractFactoryDesignPattern.Example1;

public class WindowsDialog : Dialog
{
	public override void Render()
	{
		Console.WriteLine("Rendering Windows Dialog");
	}
	public override void Show()
	{
		Console.WriteLine("Showing Windows Dialog");
	}
	public override void Close()
	{
		Console.WriteLine("Closing Windows Dialog");
	}
}

File: java/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/src/main/java/creational/abstractfactory/example1/models/concretes/WindowsDialog.java
package creational.abstractfactory.example1.models.concretes;

import creational.abstractfactory.example1.models.abstracts.Dialog;

public class WindowsDialog extends Dialog {
    public void render() { System.out.println("Rendering Windows Dialog"); }
    public void show()   { System.out.println("Showing Windows Dialog"); }
    public void close()  { System.out.println("Closing Windows Dialog"); }
}
